src/Core/Extensible.php

```diff
@@ -18,7 +17,18 @@
  */
 trait Extensible
 {
-    use CustomMethods;
+    use CustomMethods {
```

Comment:
It's not a complete solution – adding warmPropertyMethod might be better – but this shaved about 40ms https://gist.github.com/f69bad13d28e1e1ca2daf3161e4142e9

The key IMO is avoiding an additional method call.